trade-finance-manager-api/src/graphql/helpers/amendment.helpers.api-test.js

```diff
@@ -0,0 +1,292 @@
+    amendment.value = 25000;
+    amendment.currency = 'JPY';
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = 'JPY 25,000.00';
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no value', () => {
+    amendment.value = null;
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no currency', () => {
+    amendment.value = 25000;
+    amendment.currency = null;
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no currency or no value', () => {
+    amendment.value = null;
+    amendment.currency = null;
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+});
+
+describe('calculateNewFacilityValue()', () => {
+  const amendment = { currency: CURRENCY.GBP };
+
+  it('should return the same number since in GBP', () => {
+    amendment.value = 25000;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(null, amendment);
+
+    const expected = amendment.value;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return a number if different currency', () => {
+    amendment.value = 25000;
+    amendment.currency = 'JPY';
+    const exchangeRate = 7.1;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(exchangeRate, amendment);
+
+    const expected = amendment.value * exchangeRate;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no facility value or currency in amendment', () => {
+    amendment.value = null;
+    amendment.currency = null;
+    const exchangeRate = 7.1;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(exchangeRate, amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no exchange rate if currency not GBP', () => {
+    amendment.value = 25000;
+    amendment.currency = 'JPY';
+    const exchangeRate = null;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(exchangeRate, amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+});
+
+describe('calculateUkefExposure()', () => {
+  it('should return the cover percentage without rounding when not needed', () => {
+    const facilityValueInGBP = '5000';
+    const coverPercentage = 80;
+    const result = amendmentHelpers.calculateUkefExposure(facilityValueInGBP, coverPercentage);
+
+    const expected = 5000 * (80 / 100);
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return a rounded number when more than 2 d.p', () => {
```

Comment:
`d.p` please amend to `decimal point` and everywhere else.

Reply:
done